docs/using-airbyte/getting-started/oss-quickstart.md

```diff
@@ -7,361 +7,276 @@ import TabItem from "@theme/TabItem";
+    To run Airbyte in a low-resource environment (fewer than 4 CPUs), specify the `--low-resource-mode` flag to the local install command.
 
-1. Make sure that you have stopped the instance running in docker compose, this may require the following command:
+    ```bash
+    abctl local install --low-resource-mode
+    ```
 
-```
-docker compose stop
-```
+    :::note
+    If you see the warning `Encountered an issue deploying Airbyte` with the message `Readiness probe failed: HTTP probe failed with statuscode: 503`, allow installation to continue. You may need to allocate more resources for Airbyte, but installation will complete anyway. See [Suggested resources](#suggested-resources).
+    :::
 
-2. Make sure that you have the latest version of abctl by running the following command:
+Installation may take up to 15 minutes depending on your internet connection. When it completes, your Airbyte instance opens in your web browser at [http://localhost:8000](http://localhost:8000).
 
-```
-curl -LsfS https://get.airbyte.com | bash -
-```
+As long as your Docker Desktop daemon is running in the background, use Airbyte by returning to [http://localhost:8000](http://localhost:8000). If you quit Docker Desktop and want to return to Airbyte, start Docker Desktop again. Once your containers are running again, you can access Airbyte normally.
 
-3. Run abctl with the migrate flag set with the following command:
+## Part 3: Set up authentication
 
-```
-abctl local install --migrate
-```
+To access your Airbyte instance, you need a password.
 
-:::note
+1. Get your default password.
 
-If you're using a version of Airbyte that you've installed with `abctl`, you can find instructions on upgrading your Airbyte installation [here](../../operator-guides/upgrading-airbyte.md#upgrading-with-abctl).
+    ```bash
+    abctl local credentials
+    ```
 
-:::
+    This outputs something like this:
 
-## Using an EC2 Instance with abctl
+    ```shell
+    Credentials:
+    Email: user@company.example
+    // highlight-next-line
+    Password: random_password
+    Client-Id: 03ef466c-5558-4ca5-856b-4960ba7c161b
+    Client-Secret: m2UjnDO4iyBQ3IsRiy5GG3LaZWP6xs9I
+    ```
 
-This guide will assume that you are using the Amazon Linux distribution. However. any distribution that supports a docker engine should work with `abctl`. The launching and connecting to your EC2 Instance is outside the scope of this guide. You can find more information on how to launch and connect to EC2 Instances in the [Get started with Amazon EC2](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html) documentation from Amazon.
+    You can use that password to log into Airbyte, but you probably want to use your own email and password.
 
-1. Install the docker engine:
+3. Set an email.
 
-```shell
-sudo yum install -y docker
-```
+    ```bash
+    abctl local credentials --email your.name@example.com
```

Comment:
While this still works, I think you actually set your email during the initial setup now, so hopefully this is less necessary. @ian-at-airbyte would love to get your opinion on the work flow there. Lets go over it next week during our sync.

Reply:
I've updated the steps to reflect this. I think the UI-based way makes sense, and it's redundant to run the command now.